### PR TITLE
Attempt to fix pip build issues

### PR DESCRIPTION
**Pull request by steven-johnson**

### CODE REVIEW

File: .github/workflows/pip.yml
Comment (vksnk) [inline .github/workflows/pip.yml:7]: Is this commented out code needed?

**Review by vksnk — APPROVED** [review]

File: .github/workflows/pip.yml
Comment (steven-johnson) [reply .github/workflows/pip.yml:7]: It is (unfortunately) the only way I've found to test GitHub Actions (ie, by temporarily changing the trigger action to allow it to actually run before checking it in). I should have done this before requesting approval, sorry.

**Pull request merged**
